UnityProject/Assets/Scripts/HealthV2/Living/Mutations/BigStomachMutation.cs

```diff
@@ -22,13 +21,13 @@ public InBigStomachMutation(BodyPart BodyPart,MutationSO _RelatedMutationSO) : b
 			public override void SetUp()
 			{
 				var Stomach = BodyPart.GetComponent<Stomach>();
-				Stomach.StomachContents.SetMaxCapacity(99); //TODO better implementation //idk Custom thing, if it's preset custom
+				Stomach.ChangeStomachCapacity(200);
```

Comment:
Isn't this slots now?

Reply:
It changes the reagent capacity, stomachs have a pool of reagents. Thats what gets slowly put into the bloodstream or provides food. Reagents can be added by liquids (with will add to it straight), or by food, which will spill into it over time

Reply:
humm, wouldn't it be better to to give it more slots, also I imagine this might cause issues with food that gives very little substance filling you up because you have limited slots

Reply:
could give it more slots, but I think there was an issue with nonstatic fields. Ill make it do both

Reply:
Could do both? I think there was an issue when I tried that. But yeah could just make it increase slots and reagent size

Reply:
I'm just asking questions of balance, since before you could eat load of crappy food,  but there would be like 1u nutriment 6u salt type of thing , now it depends on the size of your stomach so you can eat six Packets of crisps vs the amount of u storage you have

Reply:
That is true yes, but at the same time, someone shouldnt be able to eat like 20 space wtinkies in rapid succession. Also food that is less fulliling, stuff like snacks, would be digested quickly. So the only thing this really stops you from doing is gulping them down every second